app/features-json/setup_json_controller.rb

```diff
@@ -8,5 +8,19 @@ class SetupJSONController < APIController
     get "#{HOME}/configured", authenticate: false do
       return json(Services.onboarding_service.correct_setup?)
     end
+
+    get "#{HOME}/user_details" do
+      code_hosting_service = current_user_config_service.code_hosting_service(
```

Comment:
this API should be able to be called without any configuration being done yet to the server. I haven't look deeply into this, but does this method require initial configuration?